jobs/sql/extractions/hiv_patient.sql

```diff
@@ -799,6 +801,28 @@ inner join temp_partner_status tps on tps_id =
 	where tps2.patient_id = t.patient_id 
 	order by tps2.status_datetime desc limit 1)
 set t.partner_hiv_status = tps.status;
+
+
+update temp_patient t 
+inner join  hiv.patient_identifier pi2 
+on t.patient_id =pi2.patient_id
+where pi2.identifier_type =
+(select pit.patient_identifier_type_id 
+from hiv.patient_identifier_type pit 
+where uuid = 'e26ca279-8f57-44a5-9ed8-8cc16e90e559')
+and pi2.patient_id=t.patient_id ORDER by  pi2.date_created desc limit 1
+set t.biometrics_collected=IF(pi2.date_created, 1, 0);
+
+
+update temp_patient t 
+inner join  hiv.patient_identifier pi2 
+on t.patient_id =pi2.patient_id
+where pi2.identifier_type =
+(select pit.patient_identifier_type_id 
+from hiv.patient_identifier_type pit 
+where uuid = 'e26ca279-8f57-44a5-9ed8-8cc16e90e559')
+and pi2.patient_id=t.patient_id ORDER by  pi2.date_created desc limit 1
+set t.biometrics_collection_date=pi2.date_created;
```

Comment:
@louidorjp I think you could do this in one statement.  You can set both columns in the same set statement.   

Also this will bring back the "latest" biometrics collection date.  Is that what you want?  Maybe call the column "latest_biometrics_collection_date"  ?